scipy/stats/_multivariate.py

```diff
@@ -3850,3 +3852,455 @@ def rvs(self, dim, size=1, random_state=None):
+
+    To evaluate the probability mass function of the multivariate
+    hypergeometric distribution, with a dichotomous population of size
+    :math:`10` and :math:`20`, at a sample of size :math:`12` with
+    :math:`8` objects of the first type and :math:`4` objects of the
+    second type, use:
+
+    >>> from scipy.stats import multivariate_hypergeom
+    >>> multivariate_hypergeom.pmf(x=[8, 4], m=[10, 20], n=12)
+    0.0025207176631464523
+
+    The `multivariate_hypergeom` distribution is identical to the
+    corresponding `hypergeom` distribution (tiny numerical differences
+    notwithstanding) when only two types (good and bad) of objects
+    are present in the population as in the example above. Consider
+    another example for a comparison with the hypergeometric distribution:
+
+    >>> from scipy.stats import hypergeom
+    >>> multivariate_hypergeom.pmf(x=[3, 1], m=[10, 5], n=4)
+    0.4395604395604395
+    >>> hypergeom.pmf(k=3, M=15, n=4, N=10)
+    0.43956043956044005
+
+    The functions ``pmf``, ``logpmf``, ``mean``, ``var``, ``cov``, and ``rvs``
+    support broadcasting, under the convention that the vector parameters
+    (``x``, ``m``, and ``n``) are interpreted as if each row along the last
+    axis is a single object. For instance:
+
+    >>> multivariate_hypergeom.pmf(x=[[8, 4], [3, 1]], m=[[10, 20], [10, 5]], n=[12, 4])
+    array([0.00252072, 0.43956044])
+
+    This combines the result of previous two examples in a single array.
+    In this example, the last axis of the each array ``x``, ``m``, and ``n``
+    are treated as independent objects and the ``pmf`` is exaluated over
+    each of them to generate a combined result.
+
+    Here, ``x.shape == (2, 2)``, ``m.shape == (2, 2)``, and
+    ``n.shape == (2,)`` but following the rules mentioned above, they behave
+    as if the rows ``[8, 4]`` and ``[3, 1]`` in ``x``, ``[10, 20]`` and
+    ``[10, 5]`` in ``m``, and ``12`` and ``4`` in ``n`` were two independent
+    objects, and as if we had two populations with ``x.shape = (2,)``,
+    ``m.shape = (2,)``, and ``n.shape = ()``. To obtain the individual
+    elements without broadcasting,  we would do this:
+
+    >>> multivariate_hypergeom.pmf([8, 4], m=[10, 20], n=12)
+    0.0025207176631464523
+    >>> multivariate_hypergeom.pmf([3, 1], m=[10, 5], n=4)
+    0.4395604395604395
+
+    This broadcasting also works for ``cov``, where the output objects are
+    square matrices of size ``m.shape[-1]``. For example:
+
+    >>> multivariate_hypergeom.cov(m=[[7, 9], [10, 15]], n=[8, 12])
+    array([[[ 1.05, -1.05],
+            [-1.05,  1.05]],
+           [[ 1.56, -1.56],
+            [-1.56,  1.56]]])
+
+    In this example, ``m.shape == (2, 2)`` and ``n.shape == (2,)``.
+    Following the rules above, ``m`` and ``n`` broadcast, as if ``m.shape == (2,)``.
+    Thus the result should also be of shape ``(2,)``, but since each output is
+    a :math:`2 \times 2` matrix, the result, in fact, has shape ``(2, 2, 2)``,
```

Comment:
```suggestion
```
What do you think - do all these words help explain what is going on, or do they just make it more confusing?

Reply:
It does sound quite confusing. I think this should be removed as you suggest.

Reply:
Sorry. I committed suggestions to batch and GitHub marked them as resolved. Feel free to unresolved and resolve them yourself

Reply:
Oh, yeah, I know it does that. Thanks for letting me know.